pkgs/by-name/si/sirula/package.nix

```diff
@@ -1,42 +1,33 @@
-{
-  lib,
-  fetchFromGitHub,
-  rustPlatform,
-  pkg-config,
-  gtk3,
-  gtk-layer-shell,
-}:
+{ lib, fetchFromGitHub, rustPlatform, pkg-config, gtk3, gtk-layer-shell, }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
   pname = "sirula";
-  version = "1.0.0-unstable-2023-09-02";
+  version = "1.1.0";
```

Comment:
Since this just uses a generic `version`, I think this can be made to use automatic updates with 
```
  passthru.updateScript = nix-update-script { };
```